packages/plugin-e2e/playwright.config.ts

```diff
@@ -25,6 +25,10 @@ export default defineConfig<PluginOptions>({
 
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
     trace: 'on-first-retry',
+    grafanaAPIUser: {
```

Comment:
nit: Besides a user account are there other ways to auth with the API? Maybe `grafanaAPIAuth` or `grafanaAPICredentials` would work rather than mentioning user?

Reply:
yep, 100% agree